tools/torch/trace_torchvision.py

```diff
@@ -255,8 +265,9 @@ def get_detection_input():
 
         # TODO try scripting instead of tracing
         example = torch.rand(1, 3, 224, 224)
-        script_module = torch.jit.trace(model, example)
-
+        #script_module = torch.jit.trace(model, example)
```

Comment:
Remove tracing completely? Or add as an option (but we can do that later if we need it for a specific model)

Reply:
Done.